### PR TITLE
Docs: Fix spelling in ArgTypes doc block image

### PR DESCRIPTION
**Pull request by kate-hall**
Closes #
N/A

## What I did

Adjusts the spelling of the word "principal" (previously "principle") for grammatical correctness. This image appears on the ArgTypes doc page: https://storybook.js.org/docs/api/arg-types

This image matches the original's color profile, alpha channel setting, and dimensions exactly. It also shaves off a couple of kilobytes from the file size.

#### The changes in this PR are covered in the following automated tests:

N/A

#### The changes in this PR are covered in the following automated tests:
- [ ] stories
- [ ] unit tests
- [ ] integration tests
- [ ] end-to-end tests

#### Manual testing

Use GitHub's onion skin or swipe features to see that no visual differences beyond the spelling correction exist between images.

### Documentation

N/A

- [ ] Add or update documentation reflecting your changes
- [ ] If you are deprecating/removing a feature, make sure to update
      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

## Checklist for Maintainers

- [ ] When this PR is ready for testing, make sure to add `ci:normal`, `ci:merged` or `ci:daily` GH label to it to run a specific set of sandboxes. The particular set of sandboxes can be found in `code/lib/cli/src/sandbox-templates.ts`
- [ ] Make sure this PR contains **one** of the labels below:
   <details>
     <summary>Available labels</summary>

     - `bug`: Internal changes that fixes incorrect behavior.
     - `maintenance`: User-facing maintenance tasks.
     - `dependencies`: Upgrading (sometimes downgrading) dependencies.
     - `build`: Internal-facing build tooling & test updates. Will not show up in release changelog.
     - `cleanup`: Minor cleanup style change. Will not show up in release changelog.
     - `documentation`: Documentation **only** changes. Will not show up in release changelog.
     - `feature request`: Introducing a new feature.
     - `BREAKING CHANGE`: Changes that break compatibility in some way with current major version.
     - `other`: Changes that don't fit in the above categories.
   
   </details>

### CODE REVIEW

**Comment by kate-hall** [comment]
I'm unable to add the `documentation` label on this PR since it is from a fork. Maintainers, please add!

**Review by jonniebigodes — APPROVED** [review]
@kate-hall, thank you for taking the time to put together this pull request and helping us improve the documentation by fixing this typo in our images. We appreciate it 🙏 ! I've checked, and all is good on my end. I will gladly merge this once the checklist clears.

Hope you have a great day.

Stay safe

**Pull request merged**
